### DISCUSSION TITLE
version 2.0.66

### DISCUSSION THREAD

**Discussion by naman108** [FreeTAKTeam/FreeTakServer#590] [Announcements]
## Summary
This release has improved the stability and performance of the TCP and SSL CoT services by implementing the digitalpy architecture. Both services now communicate through the ZeroMQ based Z-Manager and refactored internals. Further more our logging is now more configurable, and more effective at the Debug level.

## Installation
the suggested way to install FTS is by using our signature ZeroTouch installer
on a Ubuntu 22.04 machine type
``` wget -qO - https://raw.githubusercontent.com/FreeTAKTeam/FreeTAKHub-Installation/up-installer-to-v2/scripts/easy_install.sh | sudo bash```

1. If you want to install using Pypi
2. Use an Ubuntu 22.04 nachine
3. (optional) Install python3.11
4.  pypi as follows 
```pip install FreeTAKServer[UI]```

## What's Changed
* Remove Python 3.8 testing by @hickey in https://github.com/FreeTAKTeam/FreeTakServer/pull/567
* 573 log level insane by @naman108 in https://github.com/FreeTAKTeam/FreeTakServer/pull/579
* Move emergency api by @naman108 in https://github.com/FreeTAKTeam/FreeTakServer/pull/580
* converted tcp and ssl services to proper digitalpy implementations by @naman108 in https://github.com/FreeTAKTeam/FreeTakServer/pull/589

**Full Changelog**: https://github.com/FreeTAKTeam/FreeTakServer/compare/v2.0...v2.0.66b

<hr /><em>This discussion was created from the release <a href='https://github.com/FreeTAKTeam/FreeTakServer/releases/tag/v2.0.66'>version 2.0.66</a>.</em>

**Comment by jweaver72** [comment]
I have tried this method and it is not working. The installer runs, the FTS server runs and is open for connections, connections can be established but every other service that is related to nodered fails to load
